source/quickstart/go/gorilla.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Getting Started with Go on Aptible using the Gorilla Web Toolkit
-subtitle: Deploy your Gorilla web toolkit app on Aptible in about 5 minutes
+title: Golang Gorilla
```

Comment:
Having the language and framework in the title will help with search auto-complete.  We probably want this article included in results when someone searches just 'Golang".  Alternatively, we can add keywords to each article to the same affect.  Either way, I think the titles should be consistent, some are <LANGUAGE> <FRAMEWORK>, others are just <FRAMEWORK>